src/parser/jobs/mnk/modules/Fists.js

```diff
@@ -68,6 +78,48 @@ export default class Fists extends Module {
 		this.addHook('complete', this._onComplete)
 	}
 
+	normalise(events) {
+		for (let i = 0; i < events.length; i++) {
+			const event = events[i]
+			const abilityId = event.ability.guid
+			const startTime = this.parser.fight.start_time
```

Comment:
Minor, but you could move this outside of the loop. It's not going to change, so there's no reason to redeclare it every iteration. That, or just don't even bother to declare it as a local const since you only use it in one place anyway.

Reply:
Originally I used it more than once, forgot about it oops.